mlflow/server/js/src/utils/Utils.js

```diff
@@ -209,10 +212,15 @@ class Utils {
     if (run.source_version) {
       const versionString = shortVersion ? run.source_version.substring(0, 6) : run.source_version;
       if (run.source_type === "PROJECT") {
-        const match = run.source_name.match(Utils.getGitHubRegex());
-        if (match) {
-          const url = ("https://github.com/" + match[1] + "/" + match[2].replace(/.git/, '') +
-                     "/tree/" + run.source_version) + "/" + match[3];
+        const GitHubMatch = run.source_name.match(Utils.getGitHubRegex());
+        const BitBucketMatch = run.source_name.match(Utils.getBitBucketRegex());
```

Comment:
Nit: You should capitalize these as gitHubMatch and bitBucketMatch (camel case) to match the code style in the rest of the codebase.

Reply:
Done